src/components/Ui/Button.vue

```diff
@@ -23,7 +23,7 @@ defineProps({
 <style scoped lang="scss">
 .button {
   border: 1px solid var(--border-color);
-  background-color: transparent;
+  background-color: var(--block-bg);
```

Comment:
It looks like this in dark mode
![image](https://user-images.githubusercontent.com/15967809/134772015-055dd510-3c5b-4804-87bf-2cefd58b3e63.png)